_posts/2023-03-13-introducing-security-analytics.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "Introducing Security Analytics"
+authors:
+  - christophermoore
+date:   2023-03-13 12:15:00 -0700
+categories:
+  - releases
+meta_keywords: security analytics
+meta_description: Provides and overview of Security Analytics and how it fits in with the OpenSearch Project
```

Comment:
Update to meta keywords and description: 

**meta_keywords:** security analytics, security analytics in OpenSearch, open source SIEM tools

**meta_description:** Learn how the powerful security analytics capabilities in OpenSearch allow administrators to track and analyze activity in the cluster to combat cyber threats.